readme.md

```diff
@@ -249,6 +251,45 @@ Yields:
 }
 ```
 
+#### `Options`
+
+Configuration (optional).
+
+###### `options.getOnigurumaUrlFetch`
+
+Get a URL to the oniguruma WASM, typically used in browsers (`GetOnigurumaUrl`,
+optional).
+
+###### `options.getOnigurumaUrlFs`
+
+Get a URL to the oniguruma WASM, typically used in Node.js (`GetOnigurumaUrl`,
+optional).
+
+#### `GetOnigurumaUrl`
+
+Function to get a URL to the oniguruma WASM.
+
+> 👉 **Note**: this must currently result in a version 1 URL of
+> `onig.wasm` from `vscode-oniguruma`.  
+> ☝️ **Disclaimer**: Using this functionality may break your implementation
+if the static WASM you point to differs
+from the one released by `vscode-oniguruma`
```

Comment:
```suggestion

> ⚠️ **Danger**: when you use this functionality, your project might break at
> any time (when reinstalling dependencies), except when you make sure that
> the WASM binary you load manually is what our internally used
> `vscode-oniguruma` dependency expects.
> To solve this, you could for example use an npm script called `dependencies`
> (which runs everytime `node_modules` is changed) which copies
> `vscode-oniguruma/release/onig.wasm` to the place you want to host it.
```

How about this?
a) A break, so there are two paragraphs, two block quotes. Otherwise they run together
b) Stronger emoji/label
c) Offers a solution

Reply:
yeah sure, that works for me 👍